htlcswitch/link.go

```diff
@@ -1441,6 +1540,16 @@ func (l *channelLink) htlcManager() {
 				)
 			}
 
+		case qReq := <-l.quiescenceReqs:
+			l.quiescer.initStfu(qReq)
+
+			if err := l.quiescer.sendOwedStfu(); err != nil {
+				l.stfuFailf("%s", err.Error())
```

Comment:
think this error should be sent back to the user since it's an RPC call instead of failing the link - the user can then take w/e actions needed next.

Reply:
This isn't a user error. This error occurs if there is some kind of internal inconsistency. There are 3 situations when this occurs:

1. We try to build an stfu message when we've already sent one.
2. We try to build an stfu message while there are pending local updates
3. There is an error in trying to send the message itself

Reply:
> 2. user calls Quiesce, then an error occurs, the link stays untouched and the user gets an error in the response.

Agree with this - error out, ideally with a temporarily unavailable grpc code so that the user can retry later. 

(definitely out of scope for this PR) One day we could separate into:
1) The users asks for stfu
2) We stfu when nothing is pending

Similar to how we request a coop-close and then wait for htlcs to clear.